### DISCUSSION TITLE
draggable: '[data-gjs-type="mj-body"]' don't work for my custom component

### DISCUSSION THREAD

**Discussion by mingxin-yang** [GrapesJS/grapesjs#3989] [Q&A]
I use the `grapejs-mjml` plugin, I add a mj-link type, I want to it like `mj-section`, just draggable in mj-body, but it can't work.
When I set `draggable: '[data-gjs-type="mj-column"]'` , it can drag in to `mj-column`, but not at `mj-body` . I don't konw why.
Please help me.

```
editor.DomComponents.addType('mj-link',{
  model:{
    defaults: {
      editable:true,
      draggable: '[data-gjs-type="mj-body"]',
    },
  },
```

```
backbone.js:370 Invalid target position 
{errors: Array(2), model: r, context: 'sorter', target: undefined, level: 'warning'}
context: "sorter"
errors: (2) ['Target collection not found', 'Component not draggable, acceptable by [[data-gjs-type="mj-body"]]']
level: "warning"
model: r {cid: 'c935', attributes: {…}, _changing: false, _previousAttributes: {…}, changed: {…}, …}
target: undefined
[[Prototype]]: Object
```

@artf

**Comment by ronaldohoch** [comment]
Hello!
Why not use the correct semantic?

From: https://documentation.mjml.io/#mj-section

![image](https://user-images.githubusercontent.com/2287371/144605294-beb3aac1-94ca-494d-acc7-55b78fa77b5c.png)

**Reply by mingxin-yang** [reply]
I need a control that can be clicked and then the link can be entered in the pop-up box. The `mj-section` does not meet my needs, similar to this
https://github.com/artf/grapesjs-mjml/issues/250

**Comment by yongma16** [comment]
I also encountered this problem, please do you solve it?
